crates/sui-node/src/lib.rs

```diff
@@ -1082,32 +1080,18 @@ impl SuiNode {
 
         consensus_adapter.swap_throughput_profiler(throughput_profiler);
 
-        let consensus_handler_initializer = || {
-            ConsensusHandler::new(
-                epoch_store.clone(),
-                checkpoint_service.clone(),
-                state.transaction_manager().clone(),
-                state.db(),
-                low_scoring_authorities.clone(),
-                committee.clone(),
-                state.metrics.clone(),
-                throughput_calculator.clone(),
-            )
+        let consensus_handler_initializer = ConsensusHandlerInitializer {
```

Comment:
It's better practice to keep the fields private and construct through `new()` instead.